swift/swift.py

```diff
@@ -223,35 +233,24 @@ def destroyApp(self, name: str):
             apps.discard(app)
         app.deleteLater()
 
-    def updateFrames(self, name: str, update: bool = False):
+    def updateFrames(self, name: str):
         """Updates the frames of an app.
         
         Args:
             name: A name of the app to update its frames.
-            update: Whether the original frames will be updated or remained.
         """
         app = self._apps[name]
         info = self.appInfos[name]
         orgFrames = {dockWidget.widget(): dockWidget for dockWidget in self._dockWidgets[name]}
         newFrames = app.frames()
         orgFramesSet = set(orgFrames)
         newFramesSet = set(newFrames)
-        for frame in orgFramesSet if update else orgFramesSet - newFramesSet:
+        for frame in orgFramesSet - newFramesSet:
             dockWidget = orgFrames[frame]
             self.mainWindow.removeDockWidget(dockWidget)
             dockWidget.deleteLater()

```

Comment:
Oops, please make this a separate method, e.g., `removeFrame()`, as it its usage is the same as in `destroyApp()`.

Reply:
Oh, the removing part is so short that I couldn't notice it.